ereuse_devicehub/resources/action/models.py

```diff
@@ -1350,6 +1350,34 @@ class DataWipe(JoinedTableMixin, ActionWithMultipleDevices):
                           primaryjoin='DataWipe.document_id == DataWipeDocument.id')
 
 
+class ActionStatus(JoinedTableMixin, ActionWithMultipleTradeDocuments):
+# class ActionStatus(JoinedTableMixin, ActionWithMultipleDevices):
```

Comment:
Is it really necessary to have this line in the code?

Reply:
done